src/Sentry/Sentry.csproj

```diff
@@ -52,6 +53,7 @@
   -->
   <ItemGroup>
     <PackageReference Include="Polyfill" Version="1.23.0" PrivateAssets="all" />
+    <PackageReference Include="System.IO.MemoryMappedFiles" Version="4.3.0" />
```

Comment:
This would be a concern.  Though it appears it's not necessary.  The code still compiles if removed.